lib/data_update_scripts/20210503175027_move_summary_and_location_to_profile.rb

```diff
@@ -0,0 +1,29 @@
+module DataUpdateScripts
+  class MoveSummaryAndLocationToProfile
+    def run
+      # do we also need to delete the old data?
```

Comment:
I think we should do this in a follow-up PR once we're sure all data has been migrated properly.